packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -26,14 +26,14 @@ import Cookies from 'js-cookie'
  * @typedef {Object} Customer
  */
 
-const usidStorageKey = 'usid'
-const cidStorageKey = 'cid'
-const encUserIdStorageKey = 'enc-user-id'
-const tokenStorageKey = 'token'
-const refreshTokenRegisteredStorageKey = 'cc-nx'
-const refreshTokenGuestStorageKey = 'cc-nx-g'
-const oidStorageKey = 'oid'
-const dwSessionIdKey = 'dwsid'
+const usidStorageKey = '_usid'
+const cidStorageKey = '_cid'
+const encUserIdStorageKey = '_enc-user-id'
+const tokenStorageKey = '_token'
+const refreshTokenRegisteredStorageKey = '_cc-nx'
+const refreshTokenGuestStorageKey = '_cc-nx-g'
+const oidStorageKey = '_oid'
+const dwSessionIdKey = '_dwsid'
```

Comment:
I like this idea of keeping the original in situ and "name spacing" the old key names. If you wanted to scream at the dev to ensure this code is removed you could prefix them with "DEPRECATED_" or something like that.